docs/suspensive.org/src/pages/docs/react-query/infiniteQueryOptions.en.mdx

```diff
@@ -0,0 +1,3 @@
+import { Sandpack } from '@/components'
+
+# infiniteQueryOptions (WIP)
```

Comment:
I added infiniteQueryOptions at https://suspensive.org
This should be done soon.
I'll add this as issue